integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/ClusterUtils.java

```diff
@@ -122,5 +126,32 @@ public static boolean scaleCluster(String clusterName, String namespace, int rep
     getLogger().info("Updating replicas in cluster {0} using patch string: {1}", clusterName, patchStr);
     V1Patch patch = new V1Patch(patchStr);
     return patchClusterCustomResource(clusterName, namespace, patch, V1Patch.PATCH_FORMAT_JSON_PATCH);
-  }  
+  }
+
+  /**
+   * Add cluster to domain resource.
+   *
+   * @param clusterName name of the cluster resource
+   * @param namespace namespace
+   * @param domain domain resource object
+   * @param replicas scale to replicas
+   * @return modified domain resource object
+   */
+  public static DomainResource addClusterToDomain(String clusterName, String namespace,
+                                                                         DomainResource domain, int replicas) {
+    java.util.List<String> clusterNames = new java.util.ArrayList<>();
+    clusterNames.add(clusterName);
```

Comment:
I believe this is not needed if you remove the for loop. Anyway the method parameter only takes a single cluster not a list

Reply:
done